### PR TITLE
[docs] Add status header for awsxray receiver

### PR DESCRIPTION
**Pull request by Frapschen**
**Description:** <Describe what has changed.>
Add status header for awsxray receiver

### CODE REVIEW

File: receiver/awsxrayreceiver/README.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # AWS X-Ray Receiver
 
+| Status                   |           |
+| ------------------------ |-----------|
+| Stability                | [beta]    |
+| Supported pipeline types | traces    |
+| Distributions            | [contrib] |
```
Comment (TylerHelmuth) [inline receiver/awsxrayreceiver/README.md:7]: @willarmiros should the AWS distribution be included here?

**Review by TylerHelmuth — COMMENTED** [review]
/cc @willarmiros as code owner to confirm beta stability and distributions list.

File: receiver/awsxrayreceiver/README.md (hunk 1 above)
Comment (willarmiros) [reply receiver/awsxrayreceiver/README.md:7]: Yes please! https://aws-otel.github.io/docs/components/x-ray-receiver

Comment (Frapschen) [reply receiver/awsxrayreceiver/README.md]: @TylerHelmuth @willarmiros added.

**Review by TylerHelmuth — APPROVED** [review]

**Review by codeboten — APPROVED** [review]
@willarmiros please approve and we can get this merged 👍🏻

**Review by willarmiros — APPROVED** [review]
Thanks!

**Pull request merged**